python/samples/adk/basic/Dockerfile

```diff
@@ -0,0 +1,14 @@
+### STAGE 1: base image
+FROM localhost:5001/kagent-dev/kagent/kagent-adk:v0.5.5-95-g3f8cd1f4
+
+WORKDIR /app
+
+COPY  basic/ basic/
+COPY  pyproject.toml pyproject.toml
+COPY  README.md README.md
+COPY  .python-version .python-version
+COPY  uv.lock uv.lock
+
+RUN uv sync --locked --refresh
+
+ENTRYPOINT ["kagent-adk", "run", "basic", "--host", "0.0.0.0", "--port", "8080"]
```

Comment:
can we move the entrypoint ["kagent-adk", "run"] to the base image, and then the user just needs a `CMD ["basic"]` here (assuming we can defalt host and port and remove them)

Reply:
We can, but I'm a little worried about magic 😆